test/Compatibility/Test_CompatibilitySetter_CS/Test_CompatibilitySetter_CS.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <Platforms>x86;x64;arm64;AnyCPU</Platforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)\dev\Projections\CS\Microsoft.Windows.ApplicationModel.WindowsAppRuntime\Microsoft.Windows.ApplicationModel.WindowsAppRuntime.Projection.csproj" />
+  </ItemGroup>
+
+  <!-- MAIN TEST CONTENT -->
+  <PropertyGroup>
+    <WindowsAppSDKRuntimePatchMode>1.7.3</WindowsAppSDKRuntimePatchMode>
+    <WindowsAppSDKRuntimePatchMode2>1.8.2</WindowsAppSDKRuntimePatchMode2>
+    <!-- Specify multiple DisabledChanges, comma-separated. "None" is the only enum value
+         which always exists, so use that one a couple times plus a cast of a random int. -->
+    <WindowsAppSDKDisabledChanges>None, None, (CompatibilityChange)23456, None</WindowsAppSDKDisabledChanges>
+  </PropertyGroup>
+
+  <Import Project="$(RepoRoot)\build\NuSpecs\Microsoft.WindowsAppSDK.CompatibilitySetter.CS.targets" />
+  <Target Name="VerifyCompatibilitySetterFile" AfterTargets="CoreCompile">
+    <!-- Make sure the setter file was generated. -->
+    <Error Condition="!Exists('$(GeneratedFilesDir)\WinAppSDKCompatibilitySetter.cs')" Text="Missing file: $(GeneratedFilesDir)\WinAppSDKCompatibilitySetter.cs" />
+    <Error Condition="'$(WindowsAppSdkCompatibilityInitialize)' != 'true'" Text="WindowsAppSdkCompatibilityInitialize is incorrectly not set to true" />
+  </Target>
+  <!-- END: MAIN TEST CONTENT -->
+
+  <!-- AUTOINITIALIZER CONTENT: Mostly copied from Microsoft.WindowsAppSDK.AutoInitializer.CS.targets -->
```

Comment:
factor into a test.autoinitializer.props and import via test\directory.build.props?